test/unit/argument_parser/detail/format_help_test.cpp

```diff
@@ -58,35 +59,57 @@ std::string license_text()
     return str.substr(license_start, license_end - license_start);
 }
 
+namespace seqan3::detail
+{
+struct test_accessor
+{
+    static void set_terminal_width(seqan3::argument_parser & parser, unsigned terminal_width)
+    {
+        std::visit([terminal_width](auto & f)
+        {
+            if constexpr(std::is_same_v<decltype(f), seqan3::detail::format_help &>)
+                f.layout = seqan3::detail::format_help::console_layout_struct{terminal_width};
+        }, parser.format);
+    }
+};
+} // seqan3::detail
+
+using seqan3::detail::test_accessor;
+
 TEST(help_page_printing, short_help)
 {
     // Empty call with no options given. For seqan3::detail::format_short_help
     seqan3::argument_parser parser0{"empty_options", 1, argv0};
+    test_accessor::set_terminal_width(parser0, 80);
     parser0.info.synopsis.push_back("./some_binary_name synopsis");
     testing::internal::CaptureStdout();
     EXPECT_EXIT(parser0.parse(), ::testing::ExitedWithCode(EXIT_SUCCESS), "");
     std_cout = testing::internal::GetCapturedStdout();
-    expected = "empty_options"
-               "============="
-               "./some_binary_name synopsis"
-               "Try -h or --help for more information.";
-    EXPECT_TRUE(ranges::equal((std_cout | std::views::filter(!seqan3::is_space)),
-                              expected | std::views::filter(!seqan3::is_space)));
+
+    expected = "empty_options\n"
+               "=============\n"
+               "    ./some_binary_name synopsis\n"
+               "    Try -h or --help for more information.\n";
+    EXPECT_EQ(std_cout, expected);
 }
 
 TEST(help_page_printing, no_information)
 {
     // Empty help call with -h
     seqan3::argument_parser parser1{"test_parser", 2, argv1};
+    test_accessor::set_terminal_width(parser1, 80);
     testing::internal::CaptureStdout();
     EXPECT_EXIT(parser1.parse(), ::testing::ExitedWithCode(EXIT_SUCCESS), "");
     std_cout = testing::internal::GetCapturedStdout();
-    expected = "test_parser"
-               "===========" +
+    expected = "test_parser\n"
+               "===========\n"
+               "\n" +
                basic_options_str +
+               "\n"
+               "  \n"
```

Comment:
Seems like this is part of the `basic_option_str` no? Then you don't have to add it every time manually?

Reply:
Since the tests pass, I would say that is not part of `basic_option_str` :)

Reply:
Of course the current tests pass because you added this after every single occurrence of `basic_options_str` no matter what follows afterwards? :) 

It actually is part of it, see [here](https://github.com/seqan/seqan3/blob/master/include/seqan3/argument_parser/argument_parser.hpp#L708) (Although I would not like to point out this open todo in the code.. :speak_no_evil: ).